app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less

```diff
@@ -289,7 +289,7 @@
         text-align: center;
     }
 
-    .page-header,
+    .page-header .panel.wrapper,
```

Comment:
Hi @Flyingmana , thanks for contribution. Wouldn't it be enough to just limit the class to `.page-header .panel`?

Reply:
probably not, as there is also .panel.header occurrence one level deeper in html.

![image](https://user-images.githubusercontent.com/237319/76069934-ff85a000-5f93-11ea-80bb-18c09a63f103.png)